lib/driver.js

```diff
@@ -149,25 +156,31 @@ class BaseDriver extends MobileJsonWireProtocol {
     // automatic session deletion in this.onCommandTimeout. Of course we don't
     // want to trigger the timer when the user is shutting down the session
     // intentionally
+    let onCommandTimeout = async function () {
```

Comment:
Yes I moved it, and fixed the promise logic, there was a sync function as a a parameter of setTimeout.